pkg/apis/app/v1alpha1/conditions.go

```diff
@@ -23,19 +23,19 @@ import (
 type ConditionType string
 
 const (
-	// DeployedConditionType - The KogitoApp is deployed
+	// DeployedConditionType - The KogitoRuntime is deployed
 	DeployedConditionType ConditionType = "Deployed"
-	// ProvisioningConditionType - The KogitoApp is being provisioned
+	// ProvisioningConditionType - The KogitoRuntime is being provisioned
 	ProvisioningConditionType ConditionType = "Provisioning"
-	// FailedConditionType - The KogitoApp is in a failed state
+	// FailedConditionType - The KogitoRuntime is in a failed state
 	FailedConditionType ConditionType = "Failed"
```

Comment:
I believe we are not using these constants any more, I could be wrong, though. Could you please double check?

Reply:
`ConditionType` are used under `Condition` struct which is further used in `KogitoServiceStatus`

Reply:
done

Reply:
so should be `KogitoService` and not `KogitoRuntime` or ?